src/pkg/resourceviews/subscriptions.go

```diff
@@ -72,7 +79,7 @@ func (s *SubscriptionListView) SelectItem(subscriptionID string) {
 	}
 }
 
-func (s *SubscriptionListView) Update() error {
+func (s *SubscriptionListView) InitList(layout *layout.AppLayout) error {
```

Comment:
I'd err on not renaming this (yet anyhow) so the resourceviews remain consistent. Hopefully we'll start zeroing in on what the commonality between the different implementations and we can start pulling functionality into an interface or other abstraction. Also the layout should be available via the s.Parent variable.

Reply:
good point, just added the change